server.py

```diff
@@ -460,6 +480,26 @@ def delete(self, region, id, region_to_change):
 
         return convert_tournament_to_response(dao.get_tournament_by_id(tournament.id), dao)
 
+class FinalizeTournamentResource(restful.Resource):
+    """ Converts a pending tournament to a tournament.
+        Works only if the PendingTournament's alias_to_id_map is completely filled out.
+        Route restricted to admins for this region. """
+    def post(self, region, id):
+        dao = Dao(region, mongo_client=mongo_client)
+        user = get_user_from_access_token(request.headers, dao)
+        if not is_user_admin_for_region(user, region):
+            return 'Permission denied', 403
+
+        pending_tournament = dao.get_pending_tournament_by_id(ObjectId(id))
+        if not pending_tournament:
+            return "No pending tournament found with that id.", 400
+        
+        try:
+            tournament = Tournament.from_pending_tournament(pending_tournament)
```

Comment:
Can you add some sanity checks to `from_pending_tournament()`? Check out this: I'd like to do some sanity checks here. Check out: https://trello.com/c/CiAMXRvd/132-function-to-convert-pending-tournament-to-real-tournament